scripts/robot_log_dat2csv.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+"""Convert a binary log file into a plain text csv file."""
+import argparse
+import numpy as np
+
+import robot_interfaces
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("infile", type=str, help="Binary input file.")
+    parser.add_argument("outfile", type=str, help="Output CSV file.")
+    parser.add_argument(
+        "--number-of-fingers",
+        "-n",
+        type=int,
+        choices=[1, 3],
+        default=3,
+        help="Number of fingers of the robot.",
+    )
+    args = parser.parse_args()
+
+    if args.number_of_fingers == 1:
+        module = robot_interfaces.finger
+    else:
+        module = robot_interfaces.trifinger
+
+    log = module.LogReader(args.infile)
```

Comment:
so like BinaryLogReader (referring to my comment on the other PR) here to be more explicit, then it's clear what's happening just as the method in `trifinger_backend` is `write_current_buffer_binary`

Reply:
done